chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -204,8 +204,11 @@ pane.items.trash.multiple		= Are you sure you want to move the selected items to
 pane.items.delete.title			= Delete
 pane.items.delete				= Are you sure you want to delete the selected item?
 pane.items.delete.multiple		= Are you sure you want to delete the selected items?
-pane.items.menu.remove                    = Remove Item from Collection
-pane.items.menu.remove.multiple           = Remove Items from Collection
+pane.items.remove.title			= Remove from Collection
+pane.items.remove				= Are you sure you want to remove the selected item from the currently selected collection?
+pane.items.remove.multiple		= Are you sure you want to remove the selected items from the currently selected collection?
```

Comment:
(Fortunately no one will ever read this message, so it doesn't matter too much. And we do actually use "this" in some contexts, so I think it's fine here.)